projects/commudle-admin/src/app/feature-modules/users/components/profile-cards/user-profile-card-medium/user-profile-card-medium.component.ts

```diff
@@ -9,7 +9,7 @@ import {IUser} from 'projects/shared-models/user.model';
 export class UserProfileCardMediumComponent implements OnInit {
 
   @Input() user: IUser;
-  @Input() maxNameLength = 50;
+  @Input() maxNameLength = 20;
```

Comment:
Changing here is not an issue, but please verify if this affects other places where this component is used.
Another way is to use property binding in the lab card to specify the value for maxNameLength